templates/default/ntp.conf.erb

```diff
@@ -62,14 +62,18 @@ disable monitor
 <% ( node['ntp']['servers'] - node['ntp']['peers'] ).each do |ntpserver| -%>
 <%#   Loop through defined servers, but don't try to upstream ourself %>
 <%   if node['ipaddress'] != ntpserver and node['fqdn'] != ntpserver -%>
+<%# IPv4 %>
 <%  -%>server <%= ntpserver %><% if node['ntp']['server']['use_iburst'] -%> iburst<% end -%><% if node['ntp']['server']['use_burst'] -%> burst<% end -%> minpoll <%= node['ntp']['server']['minpoll'] %> maxpoll <%= node['ntp']['server']['maxpoll'] %>
 <%  -%>restrict <%= ntpserver %> nomodify notrap noquery
+<%# IPv6 %>
+<%  -%>server -6 <%= ntpserver %><% if node['ntp']['server']['use_iburst'] -%> iburst<% end -%><% if node['ntp']['server']['use_burst'] -%> burst<% end -%> minpoll <%= node['ntp']['server']['minpoll'] %> maxpoll <%= node['ntp']['server']['maxpoll'] %>
```

Comment:
`-6` is only used for DNS resolution. "while a -6 qualifier forces DNS resolution to the IPv6 namespace." ~http://doc.ntp.org/4.2.6/confopt.html
If `node['ntp']['servers']` contains a list of IPv4 addresses, then every server will be defined twice.